src/main/java/com/rtb/projectmanagementtool/user/CreateUserServlet.java

```diff
@@ -0,0 +1,66 @@
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.users.*;
+import java.io.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+
+/** Servlet that returns task data */
+@WebServlet("/create-user")
+public class CreateUserServlet extends HttpServlet {
+
+  DatastoreService datastore;
+  Cookie cookie;
+
+  public CreateUserServlet() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  // for testing
+  public CreateUserServlet(DatastoreService datastore) {
+    this.datastore = datastore;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // call auth service to generate AuthID
+    UserService userService = UserServiceFactory.getUserService();
+    String AuthID = userService.getCurrentUser().getUserId();
+
+    // initialize userMajors
+    ArrayList<String> majors = new ArrayList<>();
+
+    // create new UserData object with UserID
+    UserData newUser = new UserData("", "", 0, majors, Skills.NONE, 0);
+
+    // to entity
+    Entity entity = newUser.toEntity();
+
+    // put in datastore
+    datastore.put(entity);
+
+    // get userID for cookie
+    long userID = (long) entity.getKey().getId();
+
+    // set AuthID in entity
+    entity.setProperty("AuthID", AuthID);
+
+    // new AuthOps object
+    AuthOps auth = new AuthOps(datastore);
+
+    // get/create cookie and set value to userID
+    cookie = auth.getCurrCookie(request);
+    String userIDString = Long.toString(userID);
+    cookie.setValue(userIDString);
+
+    // redirect to user_settings
+    response.sendRedirect("/user_settings.html");
+
+    // now how do I update the rest of my user's fields given their input
```

Comment:
Maybe create servlets for each form in the user_settings.html page. When a user clicks "submit" in that form, it'll set a POST request to that servlet. Then, in the servlet, use `request.getParameter("parameter_name")` to get all of the user's input.

Alternatively, if you want to keep it all in one servlet, you can add a hidden input to the html that tells you which form was submitted. (Maybe there's a better way to get the form id without using a hidden input?) Then check in the servlet which form was submitted and make updates accordingly. I'd recommend making some UserController methods to update different attributes so you don't have to write all of the logic here.